nextflow.config

```diff
@@ -248,5 +248,25 @@ manifest {
     doi             = ''
 }
 
+validation {
+    help {
+        enabled = true
+        beforeText = """
+-\033[2m----------------------------------------------------\033[0m-
+                                       \033[0;34m    ///\033[0;32m/// \033[0m
+\033[0;34m        ___     __   _  _   __   __    \033[0;34m  ///\033[0;32m///// \033[0m
+\033[0;34m  |\\ | |__  __ /  ` | \\/ | / _` / _`   \033[0;34m////\033[0;32m////// \033[0m
+\033[0;34m  | \\| |       \\__, |    | \\__| \\__|   \033[0;34m///\033[0;32m///// \033[0m
+                                       \033[0;34m///\033[0;32m/// \033[0m
+\033[0;35m  ${manifest.name} ${manifest.version}\033[0m
+-\033[2m----------------------------------------------------\033[0m-
+"""
+        command = "nextflow run nf-cmgg/germline -profile docker --input <samplesheet> --outdir <output_directory>"
```

Comment:
THIS
IS
PREPROCESSING!
_insert 300 meme_

Reply:
woops

Reply:
*crawls out of the endless pit* your issue has been fixed ;)